src/main/resources/roleconfig/role_common.json

```diff
@@ -2112,9 +2141,9 @@
     ]
   },
   {
-    "name": "hmcts-ctsc",
-    "label": "HMCTS CTSC",
-    "description": "HMCTS CTSC role for ctsc users",
+    "name": "hearing-manager",
```

Comment:
hearing manager for CTSC appearing two times. one for publiclaw and another for privatelaw. can you please merge as one